runtime/onert/core/src/compiler/ShapeValidator.cc

```diff
@@ -162,11 +162,47 @@ void ShapeValidator::visit(const ir::operation::BCQGather &node)
   // more shape validation will be done inside kernel.
 }
 
+void ShapeValidator::visit(const ir::operation::Conv2D &node)
+{
+  const auto &operands = _graph.operands();
+  const auto ofm_index{node.getOutputs().at(0)};
+  if (operands.at(ofm_index).info().isDynamic())
+    return;
+
+  const auto bias_index{node.getInputs().at(ir::operation::Conv2D::Input::BIAS)};
+
+  OP_REQUIRES(!operands.exist(bias_index) || operands.at(bias_index).shape().rank() == 1);
```

Comment:
`operands.exist(bias_index)` is used for optional input, but bias is not optional input. If bias is not exists, operator node's input size is 2 (IFM, Weight). But our loader does not support size 2 input `Conv2D` and `FullyConnected`. So it does not need to check `!operands.exist(bias_index)`.

Reply:
Is the following test incorrect? The test fails without `!operands.exist(bias_index)` check so I need to either have this check or modify this test.

https://github.com/Samsung/ONE/blob/7be192529936563e0910ae903f43a9eb97fc9214/tests/nnfw_api/src/GenModelTests/one_op_tests/FullyConnected.test.cc#L152-L179

Reply:
I've checked tensorflow lite kernel and found out FullyConnected's bias can be optional tensor. Conv2d's bias is not optional. Please remain optional check for FC and remove for Conv2D.

Reply:
`!operands.exist(bias_index)` has been removed from `Conv2D`.